package.json

```diff
@@ -28,11 +28,15 @@
   ],
   "jest": {
     "collectCoverage": true,
-    "moduleFileExtensions": [
-      "js"
-    ],
-    "moduleDirectories": [
-      "node_modules"
+	  "coverageDirectory": "./coverage/",
```

Comment:
I added `coverageDirectory` to try and get the codecov ci check to succeed. Right now my best guess is that it's only not succeeding because it compares the codecov to master branch - which currently has no tests. Please advise.